src/gc/gc.cpp

```diff
@@ -9080,8 +9080,8 @@ class seg_free_spaces
             {
                 new_address += pad;
             }
-            assert ((chosen_power2 && (i == 0)) ||
-                    (!chosen_power2) && (i < free_space_count));
+            assert (((chosen_power2 && (i == 0))) ||
```

Comment:
There is one extra unnecessary pair of () on this line

Reply:
removed them